ios/FluentUI.Demo/FluentUI.Demo/Demos/ActivityIndicatorDemoController.swift

```diff
@@ -235,6 +235,13 @@ class ActivityIndicatorDemoController: DemoTableViewController {
 
     @objc private func startStopActivity() {
         isAnimating.toggle()
+
+        guard let section: Int = ActivityIndicatorDemoSection.allCases.firstIndex(of: .settings),
+              let row: Int = ActivityIndicatorDemoSection.settings.rows.firstIndex(of: .startStopActivity) else {
```

Comment:
Nit: we avoid declaring types for inline variables unless strictly needed. See the specific reference in our style guide at [Type Inference | swift-guide (microsoft.github.io)](https://microsoft.github.io/swift-guide/TypeInference.html)

Reply:
Based on the linked guide, it would still make sense to declare types in such cases. Here `Int` is not directly used as literal. It is the result of a function `firstIndex` on another type. So for quick understanding, I think it is helpful to have the type `Int` specified here.

Reply:
The guide is consistent on this point:

> Use type annotation as little as possible when declaring a constant or a variable; instead let the compiler infer the variable type.

> Let the compiler infer the type of a constant or variable whenever possible.

The primary exception to this policy is for stored properties; local variables should almost never need a type declared, particularly when they are storing the result of a well-understood method like `firstIndex`.